src/components/Calendar/Calendar.js

```diff
@@ -219,7 +219,7 @@ export default function CalendarView() {
     const academicEventsDisplay = (
       <> <EventCard justifyContent='auto' width='92vw' height='30px' marginTop='5px' overflow='initial'
         content={<EventHeader content={"Important Academic Events"}/>}  backgroundColor='#E5A712' />
-       <div className="events">
+       <div className="events" style={{overflowY:'auto'}}>
```

Comment:
Why is overflowY set here when it is already set for the events css class? What does that do?

Reply:
apparently nothing. I removed it and behaviour seems the same